tools/topology/sof/pipe-simple-playback.m4

```diff
@@ -0,0 +1,26 @@
+# Low Latency Passthrough Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+#  B0 --> sink DAI0
```

Comment:
can we rename this to pipe-dai-playback.m4

Reply:
> can we rename this to pipe-dai-playback.m4

There is another pipe-dai-playback.m4 in the same folder already.

Reply:
pipe-dai-endpoint